### PR TITLE
Remove support for Image Profile

### LINKED ISSUE #63: At risk: support for IMSC Image Profile
This feature is expected to be removed before advancement to Proposed Recommendation if sufficient implementation experience cannot be demonstrated.

### PR DESCRIPTION
**Pull request by palemieux**
Closes #63

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/imsc-hrm/pull/73.html" title="Last updated on Dec 31, 2023, 4:39 AM UTC (0f0c423)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/imsc-hrm/73/1f16058...0f0c423.html" title="Last updated on Dec 31, 2023, 4:39 AM UTC (0f0c423)">Diff</a>

### CODE REVIEW

File: spec/imsc-hrm.html [hunk 1]
```diff
@@ -550,19 +537,13 @@ <h2>Algorithm</h2>
...
 
-        <li>BDraw is the normalized background drawing performance factor;</li>
+        <li>BDraw is the normalized background drawing performance factor; and</li>
 
         <li>DUR<sub>T</sub>(E<sub>n</sub>) is the duration, in seconds, for painting the text subtitle content for <a>Intermediate
-        Synchronic Document</a> E<sub>n</sub>, as specified in Section <a href="#paint-text"></a>; and
+        Synchronic Document</a> E<sub>n</sub>, as specified in Section <a href="#paint-text"></a>.
         </li>
 
-        <li>DUR<sub>I</sub>(E<sub>n</sub>) is the duration, in seconds, for painting the image subtitle content for <a>Intermediate
```
Comment (nigelmegitt) [inline spec/imsc-hrm.html:559]: DUR<sub>I</sub> is still present in the equation above.

File: spec/imsc-hrm.html [hunk 2]
```diff
@@ -1060,6 +948,19 @@ <h3>Changes since the <a href="https://www.w3.org/TR/2021/WD-imsc-hrm-20211109/"
...
       </dl>
     </section>
+
+    <section>
+      <h3>Changes since the <a href="https://www.w3.org/TR/2023/CRD-imsc-hrm-20231220/">first Candidate Recommendation</a></h3>
+      <dl>
+        <dt>Removed support for Image Profile</dt>
+        <dd>
+          <p>Support for IMSC Image Profile, which was at feature as-risk, was removed as not sufficient implementation
+          experience could be demonstrated.</p>
```
Comment (nigelmegitt) [inline spec/imsc-hrm.html:958]: Reads a bit strangely, just a grammatical nit:

```suggestion
          <p>Support for IMSC Image Profile, which was an at-risk feature, was removed due to insufficient demonstrable
          implementation experience.</p>
```

File: spec/imsc-hrm.html [hunk 3]
```diff
@@ -205,10 +202,6 @@
         <li>at least 1 content producing implementation and at least 1 validating implementation; or</li>
         <li>at least 2 independent validating implementations.</li>
       </ul>
-      <p>The Working Group has identified the following <dfn data-cite="w3c-process/#at-risk">at
-      risk</dfn> features:</p>
-
-      <div class="issue" data-number="63"></div>
```
Comment (nigelmegitt) [inline spec/imsc-hrm.html:211]: Is it worth replacing this with an explicit statement that there are no at-risk features? It's a bit strange to have a lone "At risk features may be removed..." sentence without explaining that there aren't any.

**Review by nigelmegitt — CHANGES_REQUESTED** [review]
Need to fix equation; a couple of other more minor nits/questions.

File: spec/imsc-hrm.html (hunk 3 above)
Comment (palemieux) [reply spec/imsc-hrm.html:211]: I looked at IMSC 1.2 CR, which did not have at-risk features, and could not find any mention of them, so I think we should just remove the sentence "At risk features may be removed...".

Comment (himorin) [reply spec/imsc-hrm.html:211]: I believe this is up to editors on at-risk text, whether both to remove following line for possible removal of at-risk feature, and to have an explicit statement with no at-risk...

(bikeshed has `At Risk` keyword, and all of at-risk related text will appear only when `At Risk` keyword is included in its source.)

**Comment by palemieux** [comment]
@nigelmegitt @himorin Ready for your review.

File: spec/imsc-hrm.html [hunk 4]
```diff
@@ -189,7 +186,7 @@
 <body>
   <section id='abstract'>
     <p>This specification specifies an Hypothetical Render Model (HRM) that constrains the presentation complexity of
-    documents that conform to any of the TTML Profiles for Internet Media Subtitles and Captions ([[IMSC]]).</p>
+    documents that conform to the Text Profile specified in Internet Media Subtitles and Captions ([[IMSC]]).</p>
```
Comment (nigelmegitt) [inline spec/imsc-hrm.html:189]: We lost the sense of support across versions of IMSC. This essentially duplicates the term definition of "IMSC Document Instance".

```suggestion
    documents that conform to the Text Profiles specified in any edition of Internet Media Subtitles and Captions ([[IMSC]]).</p>
```

File: spec/imsc-hrm.html [hunk 5]
```diff
@@ -553,14 +538,8 @@ <h2>Algorithm</h2>
         Synchronic Document</a> E<sub>n</sub>, as specified in Section <a href="#paint-text"></a>; and
```
Comment (nigelmegitt) [inline spec/imsc-hrm.html:538]: Need to move the trailing "and" to the previous bullet.

**Review by nigelmegitt — APPROVED** [review]
Approving modulo minor editorial change proposals.

**Review by himorin — APPROVED** [review]
Seems good for me.
Also I've checked mention on profiles and 'image' over whole text, and found no issue.

**Comment by nigelmegitt** [comment]
(as Chair of TTWG) Extended review period for this change has now ended, this can be merged.

**Pull request merged**